amida-tech/amida-auth-microservice
# Feature/add UUID v2

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
1. Adds migrations (already approved, in v2.3.0 per @mountHouli instead of develop)
2. Fixes bug in package.json that contained "false" instead of "^"
3. Fixes refreshToken usage so tests pass 
4. Includes jwtToken in returned body on account creation, so those calling endpoints to create account do not need to immediately make a call to sign in the account right after (helps reduce redundancy, used in messaging changes)
5. update yarn test command to use .env.test

#### Related JIRA tickets:
https://jira.amida.com/browse/DEVOPS-307
https://jira.amida.com/browse/ORANGE-910

#### How should this be manually tested?
Set up .env to enable refresh token and account creation
`yarn`
`yarn migrate`
`yarn start`
create an account, verify jwtToken is returned to the response body
setup .env.test
run `yarn test` to verify tests
